src/app/parcels/ParcelsPage.tsx

```diff
@@ -434,31 +434,40 @@ const ParcelsPage: React.FC<{}> = () => {
 
     const parcelsTableFetchAbortController = useRef<AbortController | null>(null);
 
-    useEffect(() => {
-        if (parcelId) {
-            setSelectedParcelId(parcelId);
-            setModalIsOpen(true);
-        }
-    }, [parcelId]);
-
-    useEffect(() => {
-        const fetchAndSetClientIdForSelectedParcel = async (): Promise<void> => {
+    const fetchAndSetClientIdForSelectedParcel = async (): Promise<void> => {
+        if (!parcelId) {
```

Comment:
why do we check this? (the logic around here will change from Michael's bug fix soon though https://github.com/NorwoodAndBrixtonFoodbank/nbf-website/pull/226)

Reply:
I think it kept trying to run this function when no parcels are being selected and error messages kept coming up